File: C++/Iniciante/1013.cpp
#include <iostream>
#include <iomanip> 
#include <cmath>
using namespace std;
 
int main(){
    int a, b, c, MaiorAB;
		cin >> a >> b >> c;
	
	MaiorAB = (a+b+abs(a-b))/2;
	MaiorAB = (MaiorAB+c+abs(MaiorAB-c))/2;
	
    cout << MaiorAB << " eh o maior" << endl;
    return 0;
}//fim do algoritmo

File: Python3/Iniciante/1013.py
# -*- encoding: utf-8 -*-

ENTRADA = input()

A = int(ENTRADA.split()[0])
B = int(ENTRADA.split()[1])
C = int(ENTRADA.split()[2])

MAIORAB = (A+B+abs(A-B))/2
MAIOR = (MAIORAB+C+abs(MAIORAB-C))/2

print("%d eh o maior" % MAIOR)
